### ISSUE TITLE
[Bug] Taskcluster failure on old_decision push() takes no arguments 

### ISSUE THREAD

**Issue by rpappalax** [mozilla-mobile/fenix#5369]
Taskcluster job is failing on:

```
﻿k 2019-09-16T16:54:28.616Z]   File "taskcluster/fenix_taskgraph/loader/old_decision.py", line 52, in loader
[task 2019-09-16T16:54:28.617Z]     ordered_groups_of_tasks = push(builder)
[task 2019-09-16T16:54:28.617Z] TypeError: push() takes no arguments (1 given)
[taskcluster 2019-09-16 16:54:28.896Z] === Task Finished ===
[taskcluster 2019-09-16 16:54:30.424Z] Unsuccessful task run with exit code: 1 completed in 26.541 seconds
```

**Comment by rpappalax** [comment]
When I remove the build arg from old_decision.py I get an upstream error in /builds/worker/checkouts/src/automation/taskcluster/decision_task.py

```
[task 2019-09-16T20:34:17.535Z] Generating full task set
[task 2019-09-16T20:34:17.557Z] Error loading tasks for kind old-decision:
[task 2019-09-16T20:34:17.557Z] Traceback (most recent call last):
[task 2019-09-16T20:34:17.557Z]   File "/builds/worker/.local/lib/python2.7/site-packages/taskgraph/generator.py", line 261, in _run
[task 2019-09-16T20:34:17.557Z]     new_tasks = kind.load_tasks(parameters, list(all_tasks.values()))
[task 2019-09-16T20:34:17.557Z]   File "/builds/worker/.local/lib/python2.7/site-packages/taskgraph/generator.py", line 75, in load_tasks
[task 2019-09-16T20:34:17.557Z]     for task_dict in transforms(trans_config, inputs)]
[task 2019-09-16T20:34:17.557Z]   File "taskcluster/fenix_taskgraph/loader/old_decision.py", line 52, in loader
[task 2019-09-16T20:34:17.557Z]     ordered_groups_of_tasks = push()
[task 2019-09-16T20:34:17.557Z]   File "/builds/worker/checkouts/src/automation/taskcluster/decision_task.py", line 52, in push
[task 2019-09-16T20:34:17.557Z]     return pr()
[task 2019-09-16T20:34:17.557Z] TypeError: pr() takes exactly 1 argument (0 given)
[task 2019-09-16T20:34:17.558Z] Traceback (most recent call last):
[task 2019-09-16T20:34:17.558Z]   File "/builds/worker/.local/lib/python2.7/site-packages/taskgraph/main.py", line 356, in main
[task 2019-09-16T20:34:17.558Z]     args.command(vars(args))
[task 2019-09-16T20:34:17.558Z]   File "/builds/worker/.local/lib/python2.7/site-packages/taskgraph/main.py", line 301, in decision
[task 2019-09-16T20:34:17.558Z]     taskgraph_decision(options)
[task 2019-09-16T20:34:17.558Z]   File "/builds/worker/.local/lib/python2.7/site-packages/taskgraph/decision.py", line 84, in taskgraph_decision
[task 2019-09-16T20:34:17.558Z]     full_task_json = tgg.full_task_graph.to_json()
[task 2019-09-16T20:34:17.558Z]   File "/builds/worker/.local/lib/python2.7/site-packages/taskgraph/generator.py", line 149, in full_task_graph
[task 2019-09-16T20:34:17.559Z]     return self._run_until('full_task_graph')
[task 2019-09-16T20:34:17.559Z]   File "/builds/worker/.local/lib/python2.7/site-packages/taskgraph/generator.py", line 338, in _run_until
[task 2019-09-16T20:34:17.559Z]     k, v = self._run.next()
[task 2019-09-16T20:34:17.559Z]   File "/builds/worker/.local/lib/python2.7/site-packages/taskgraph/generator.py", line 261, in _run
[task 2019-09-16T20:34:17.560Z]     new_tasks = kind.load_tasks(parameters, list(all_tasks.values()))
[task 2019-09-16T20:34:17.560Z]   File "/builds/worker/.local/lib/python2.7/site-packages/taskgraph/generator.py", line 75, in load_tasks
[task 2019-09-16T20:34:17.560Z]     for task_dict in transforms(trans_config, inputs)]
[task 2019-09-16T20:34:17.560Z]   File "taskcluster/fenix_taskgraph/loader/old_decision.py", line 52, in loader
[task 2019-09-16T20:34:17.560Z]     ordered_groups_of_tasks = push()
[task 2019-09-16T20:34:17.560Z]   File "/builds/worker/checkouts/src/automation/taskcluster/decision_task.py", line 52, in push
[task 2019-09-16T20:34:17.560Z]     return pr()
[task 2019-09-16T20:34:17.560Z] TypeError: pr() takes exactly 1 argument (0 given)
[taskcluster 2019-09-16 20:34:17.834Z] === Task Finished ===
[taskcluster 2019-09-16 20:34:19.070Z] Unsuccessful task run with exit code: 1 completed in 30.424 seconds

````

**Comment by JohanLorenzo** [comment]
Fixed in #5373. Thank you @rpappalax for letting me know!
